devops/Makefile

```diff
@@ -462,102 +396,48 @@ create-qa-api-db:
-	@oc -n $(BUILD_NAMESPACE) start-build bc/$(FORMSFLOW_BPM_NAME) --wait
-
-build-forms-flow-api:
-	test -n "$(BUILD_NAMESPACE)"
-	test -n "$(FORMSFLOW_API_NAME)"
+	test -n "$(FORMS_BUILD_TEMPLATE_PATH)"
+	test -n "$(SOURCE_REPOSITORY_URL)"
 	test -n "$(SOURCE_REPOSITORY_REF)"
-	test -n "$(SOURCE_CONTEXT_DIR)"
-	test -n "$(FORM_AI_BUILD_TEMPLATE_PATH)"
-	@echo "+\n++ BUILDING $(FORMSFLOW_API_NAME)"
-	@echo "+\n++ BUILDING $(FORMSFLOW_API_NAME) DIR: $(SOURCE_CONTEXT_DIR)$(FORMSFLOW_API_NAME) BuildFile:$(FORM_AI_BUILD_TEMPLATE_PATH)\n+"
-	@oc -n $(BUILD_NAMESPACE) process -f $(FORM_AI_BUILD_TEMPLATE_PATH) -p NAME=$(FORMSFLOW_API_NAME) -p TAG=$(BUILD_TAG) -p FORMSFLOW_SOURCE_REPO_URL=$(FORMSFLOW_SOURCE_REPO_URL) -p FORMSFLOW_SOURCE_REPO_BRANCH=$(FORMSFLOW_SOURCE_REPO_BRANCH) -p SOURCE_REPOSITORY_REF=$(SOURCE_REPOSITORY_REF) -p SOURCE_CONTEXT_DIR=$(SOURCE_CONTEXT_DIR)$(FORMSFLOW_API_NAME)| oc -n $(BUILD_NAMESPACE) apply -f -
-	@oc -n $(BUILD_NAMESPACE) start-build bc/$(FORMSFLOW_API_NAME) --wait
-
-build-forms-flow-web:
-	test -n "$(BUILD_NAMESPACE)"
-	test -n "$(FORMSFLOW_WEB_NAME)"
-	test -n "$(SOURCE_REPOSITORY_REF)"
-	test -n "$(SOURCE_CONTEXT_DIR)"
-	test -n "$(FORM_AI_BUILD_TEMPLATE_PATH)"
-	@echo "+\n++ BUILDING $(FORMSFLOW_WEB_NAME)"
-	@echo "+\n++ BUILDING $(FORMSFLOW_WEB_NAME) DIR: $(SOURCE_CONTEXT_DIR)$(FORMSFLOW_WEB_NAME) BuildFile:$(FORM_AI_BUILD_TEMPLATE_PATH)\n+"
-	@oc -n $(BUILD_NAMESPACE) process -f $(FORM_AI_BUILD_TEMPLATE_PATH) -p NAME=$(FORMSFLOW_WEB_NAME) -p TAG=$(BUILD_TAG) -p FORMSFLOW_SOURCE_REPO_URL=$(FORMSFLOW_SOURCE_REPO_URL) -p FORMSFLOW_SOURCE_REPO_BRANCH=$(FORMSFLOW_SOURCE_REPO_BRANCH) -p SOURCE_REPOSITORY_REF=$(SOURCE_REPOSITORY_REF) -p SOURCE_CONTEXT_DIR=$(SOURCE_CONTEXT_DIR)$(FORMSFLOW_WEB_NAME)| oc -n $(BUILD_NAMESPACE) apply -f -
-	@oc -n $(BUILD_NAMESPACE) start-build bc/$(FORMSFLOW_WEB_NAME) --wait
+	test -n "$(FORMIO_SOURCE_REPO_URL)"
+	test -n "$(FORMIO_SOURCE_REPO_TAG)"
+	@echo "+\n++ BUILDING $(FORMS_NAME)"
+	@echo "+\n++ BUILDING $(FORMS_NAME) DIR: $(SOURCE_CONTEXT_DIR)$(FORMS_NAME) BuildFile:$(FORMS_BUILD_TEMPLATE_PATH) BUILD_TAG:$(BUILD_TAG)\n+"
+	@oc -n $(BUILD_NAMESPACE) process -f $(FORMS_BUILD_TEMPLATE_PATH) -p NAME=$(FORMS_NAME) -p TAG=$(BUILD_TAG) -p FORMIO_SOURCE_REPO_URL=$(FORMIO_SOURCE_REPO_URL) -p FORMIO_SOURCE_REPO_TAG=$(FORMIO_SOURCE_REPO_TAG) -p SOURCE_REPOSITORY_URL=$(SOURCE_REPOSITORY_URL) -p SOURCE_REPOSITORY_REF=$(SOURCE_REPOSITORY_REF) -p SOURCE_CONTEXT_DIR=$(SOURCE_CONTEXT_DIR)$(FORMS_NAME)| oc -n $(BUILD_NAMESPACE) apply -f -
+	@oc -n $(BUILD_NAMESPACE) start-build bc/$(FORMS_NAME) --wait
 
 deploy-ha-mongo:
```

Comment:
Can we please move `deploy-ha-mongo` before the `build-forms` to leave `build-forms` and `deploy-forms` together?